src/app/routes/app-routes.tsx

```diff
@@ -83,151 +202,40 @@ function useAppRoutes() {
-      />
-      {rpcSendTransferRoutes}
-      <Route
-        path={RouteUrls.RpcSignBip322Message}
-        lazy={async () => {
-          const { RpcSignBip322MessageRoute } = await import(
-            '@app/pages/rpc-sign-bip322-message/rpc-sign-bip322-message'
-          );
-          return { Component: RpcSignBip322MessageRoute };
-        }}
-      />
-      <Route
-        path={RouteUrls.RpcSignPsbt}
-        element={
-          <AccountGate>
-            <RpcSignPsbt />
-          </AccountGate>
-        }
-      />
-    </>
-  );
-
   return createHashRouter(
     createRoutesFromElements(
-      <Route path={RouteUrls.Container} element={<Container />}>
+      <Route element={<Container />}>
         <Route path={RouteUrls.RequestDiagnostics} element={<AllowDiagnosticsPage />} />
-
         <Route
-          path={RouteUrls.Home}
+          path={`${RouteUrls.Home}*`}
           element={
             <AccountGate>
               <Home />
             </AccountGate>
           }
         >
-          <Route index element={<AssetsList />} />
-          <Route path={RouteUrls.Activity} element={<ActivityList />} />
-
-          {requestBitcoinKeysRoutes}
-          {requestStacksKeysRoutes}
-          <Route path={RouteUrls.RetriveTaprootFunds} element={<RetrieveTaprootToNativeSegwit />} />
-
-          <Route path={RouteUrls.IncreaseStxFee} element={<IncreaseStxFeeDrawer />}>
-            {ledgerStacksTxSigningRoutes}
-          </Route>
-          <Route path={RouteUrls.IncreaseBtcFee} element={<IncreaseBtcFeeDrawer />} />
-          <Route path={RouteUrls.IncreaseFeeSent} element={<IncreaseFeeSentDrawer />} />
-
-          <Route path={RouteUrls.Receive} element={<ReceiveModal />} />
-          <Route path={RouteUrls.ReceiveCollectible} element={<ReceiveCollectibleModal />} />
-          <Route
-            path={RouteUrls.ReceiveCollectibleOrdinal}
-            element={<ReceiveCollectibleOrdinal />}
-          />
-          <Route path={RouteUrls.ReceiveStx} element={<ReceiveStxModal />} />
-          <Route path={RouteUrls.ReceiveBtc} element={<ReceiveBtcModal />} />
+          {/* Modal routes overlaid on home need to be nested here with relative paths to open in new tabs */}
 
-          <Route path={RouteUrls.SendOrdinalInscription} element={<SendInscriptionContainer />}>
-            <Route index element={<SendInscriptionForm />} />
-            <Route
-              path={RouteUrls.SendOrdinalInscriptionChooseFee}
-              element={<SendInscriptionChooseFee />}
-            />
-            <Route
-              path={RouteUrls.SendOrdinalInscriptionReview}
-              element={<SendInscriptionReview />}
-            />
-            <Route
-              path={RouteUrls.SendOrdinalInscriptionSent}
-              element={<SendInscriptionSummary />}
-            />
-            <Route path={RouteUrls.SendOrdinalInscriptionError} element={<BroadcastError />} />
-          </Route>
+          {/* {receiveRoutes} */}
```

Comment:
`receiveRoutesArray` map here is just a demo of how we could use routes split out to separate files